client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -141,7 +141,7 @@ class NotificationSubscriptions extends Component {
 						</FormSectionHeading>
 						<p>
 							{ this.props.translate(
-								'{{readerLink}}Visit the Reader{{/readerLink}} to adjust individual site subscriptions.',
+								'To manage individual site subscriptions, {{readerLink}}go to the Reader{{/readerLink}}.',
```

Comment:
We could also use `hasTranslation` or `hasEnTranslation` instead of waiting for the week to get translations but I don't think it's worth couple PRs, and easier to just wait.